packages/babel-helpers/src/helpers/using.js

```diff
@@ -0,0 +1,19 @@
+/* @minVersion 7.0.0-beta.0 */
+
+export default function _using(stack, value, isAwait) {
+  if (value !== null && value !== void 0) {
```

Comment:
Maybe use `value != null`?

Unrelated to this PR: Why do we always use `===` in helpers? `==` seems to be more compatible and shorter.🤔

Reply:
It's because of `document.all` (`document.all == null`, nut it's neither `null` nor `undefined`). We have an assumption option to use `==` instead of checking with both `null` and `undefined` (and maybe we could even consider enabling that assumption by default), but assumptions do not affect the code used in the helper.